src/styles/common/_mixins.scss

```diff
@@ -172,3 +172,10 @@
 @mixin doughnut-chart-stroke($color) {
     stroke: $color;
 }
+
+@mixin breadcrumb-item($color) {
+    color:$color;
+    font:$dell-font;
+    font-size:0.7rem;
```

Comment:
It seems font-size is not matching DDS for unselected and selected item, its 0.875 and 1.5 rem resp as per spec, and as per the examples in DDS the prominent selected breadcrumb is displayed on new line, could you please check if that's possible and if it matches with wireframes?

Reply:
updated the font-size as per DDS.As per DDS there are more variant of Breadcrumb like displaying active element on next line but as per Atlantic we are requiring only basic Breadcrumb which is also present in DDS. If someone encountered new type, can create the new variant based on priority.